manual/manual/Makefile

```diff
@@ -47,13 +47,17 @@ html: htmlman/libref/style.css etex-files
 	    manual.hva -e macros.tex ../manual.tex \
 	  && $(HACHA) -tocter manual.html
 
-htmlman/libref/style.css: style.css $(STDLIB_MLIS) | unprefix_stdlib_for_ocamldoc
+htmlman/libref/style.css: style.css $(STDLIB_DEPS)
 	mkdir -p htmlman/libref
-	$(OCAMLDOC) -colorize-code -sort -html \
+	cp $(SRC)/stdlib/doc_for_pervasives $(SRC)/stdlib/pervasives.mli
+	$(OCAMLDOC)-colorize-code -sort -html \
```

Comment:
Nitpicking: this looked better with a space between `$(OCAMLDOC)` and `-colorize-code`. Without the space, it looks like the command is something like `ocamldoc-colorize-code`

Reply:
Agreed.